source/tyr/tyr/default_settings.py

```diff
@@ -27,21 +28,21 @@
 CITIES_OSM_FILE_PATH = os.getenv('TYR_CITIES_OSM_FILE_PATH', '.')
 
 # Path to the directory where the configuration file of each instance of ed are defined
-INSTANCES_DIR = '.'
+INSTANCES_DIR = os.getenv('TYR_INSTANCES_DIR', '.')
 
 # Path to the directory where the data sources for autocomplete are stocked
-TYR_AUTOCOMPLETE_DIR = "/srv/ed/autocomplete"
+AUTOCOMPLETE_DIR = os.getenv('TYR_AUTOCOMPLETE_DIR', "/srv/ed/autocomplete")
```

Comment:
Is changing the name of this variable retro-compatible with the current deployment mechanism ? 
I am wondering if  fabric or something else is setting this variable. If this is the case, this change will break the deployment.